src/celengine/galaxy.cpp

```diff
@@ -128,192 +371,115 @@ void Galaxy::setDetail(float d)
 }
 
 
-void Galaxy::setCustomTmpName(const string& tmpNameStr)
+void Galaxy::setCustomTmpName(std::string&& tmpNameStr)
 {
-    if (customTmpName == nullptr)
-        customTmpName = new string(tmpNameStr);
-    else
-        *customTmpName = tmpNameStr;
+    customTmpName = std::move(tmpNameStr);
```

Comment:
this change is not equal. with it every `Galaxy` object will be 24 bytes bigger (on amd64)

Reply:
Turns out we don't actually need this field at all, I'm going to get rid of it.